testdata/picard/quality/chrM.empty.interval_list

```diff
@@ -0,0 +1,4 @@
+@HD	VN:1.0	SO:coordinate
+@SQ	SN:chrM	LN:16903	AS:mm9	UR:ftp://hgdownload.cse.ucsc.edu/goldenPath/Mus_musculus_assembly9/bigZips/chromFa.tar.gz	M5:11c8af2a2528b25f2c080ab7da42edda	SP:Mus	musculus
```

Comment:
Given that the reference we actually use is not the one listed in the ftp site, could you change this? (below as well)